apps/admin-gui-e2e/src/integration/resource-management-perun_admin.spec.js

```diff
@@ -0,0 +1,150 @@
+context('Actions', () => {
+  const dbVoName = 'test-e2e-vo-from-db-3';
+  const dbResourceName = 'test-e2e-resource-from-db-2';
+
+  const dbAddManager = 'test4';
+  const dbRemoveManager = 'resource-admin-2';
+
+  const dbAttributeToAdd = 'user-settings-name';
+  const dbAttributeToDelete = 'user-settings-description';
+
+  const dbGroupToAssign = 'test-e2e-group-from-db-2';
+  const dbGroupToRemove = 'test-e2e-group-from-db-1';
+
+  before(() => {
+    if (Cypress.env('BA_USERNAME_RESOURCE_MANAGER')) {
+      sessionStorage.setItem('baPrincipal', '{"name": "resourceManager"}');
+      sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME_RESOURCE_MANAGER'));
+      sessionStorage.setItem('basicPassword', Cypress.env('BA_PASSWORD_RESOURCE_MANAGER'));
+      cy.visit('service-access');
+    }
+  });
+
+  beforeEach(() => {
+    cy.visit('organizations')
+      .get(`[data-cy=${dbVoName}]`)
+      .click()
+      .get('[data-cy=resources]')
+      .click()
+      .get('[data-cy=resource-list]')
+      .click()
+      .get(`[data-cy=${dbResourceName}]`)
+      .click()
+  })
+
+  it('test add attribute', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/**').as('setAttributes')
+      .get('[data-cy=attributes]')
+      .click()
+      .get('[data-cy=add-attributes]')
+      .click()
+      .get(`[data-cy=${dbAttributeToAdd}-value]`)
+      .type('test')
+      .intercept('**/attributesManager/getAttributes/vo**').as('getAttributes')
```

Comment:
The correct call which is using here is with `r` attribute instead of `vo`.
```suggestion
      .intercept('**/attributesManager/getAttributes/r**').as('getAttributes')
```